cicd/rrp-pool.Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM rust:1.82 AS build
+
+RUN apt-get update && apt-get install -y musl-tools
+
+WORKDIR /stratum
+COPY ./ .
+
+WORKDIR /stratum/roles/pool
+RUN rustup target add x86_64-unknown-linux-musl
```

Comment:
Ensure that 'rustup target add' is followed by necessary checks to verify the target was added successfully.